lib/src/widgets/chat/typing/ds_dot_animation.widget.dart

```diff
@@ -0,0 +1,150 @@
+import 'package:flutter/material.dart';
+
+import 'ds_dot.widget.dart';
+
+/// Creates a list of animated dots for use in typing identification and other situations
+///
+/// To define the stitch appearance use [size] and [color]
+/// Use [numberOfDots] to set the amount of dots and use [innerPadding] to set the spacing between them
+/// To change the animation speed use [animationTime]
+/// To set the dot ascent level use [upLevelAnimation]
+class DSDotAnimation extends StatefulWidget {
+  final double size;
+  final Color color;
+  final int numberOfDots;
+  final double innerPadding;
```

Comment:
I think that the 'inner' word is useless in the context of padding, mainly because 'padding' already emphasize that it'll add padding to its child.

Reply:
Done